docs/UserGuide.md

```diff
@@ -261,8 +261,11 @@ Format: `find c/COMPANY | r/ROLE | s/STAGE`
 
 
 Examples:
-* `find r/Engineer` returns `Software Engineer` and `Backend Engineer`.
-* `find s/Interview` returns `Technical Interview` and `Behavioural Interview`.
+*`find r/Engineer` returns `Software Engineer` and `Backend Engineer`:
+  |       Before        |       After      |
+  |:-------------------------:|:-------------------------:|
+  |![before layout](images/BeforeFind.png) |![after layout](images/AfterFind.png) |
+
```

Comment:
![Screenshot 2022-11-06 at 2 15 52 PM](https://user-images.githubusercontent.com/77665182/200157129-ebdccfb1-f4e2-4498-9e2a-c5c27b48defd.png)

formatting error here, can be fixed with:
```suggestion
* `find r/Engineer` returns `Software Engineer` and `Backend Engineer`:

  |       Before        |       After      |
  |-------------------------|-------------------------|
  |![before layout](images/BeforeFind.png) |![after layout](images/AfterFind.png) |
  
```
which will give u this, but the images aren't really visibile imo
![Screenshot 2022-11-06 at 2 18 58 PM](https://user-images.githubusercontent.com/77665182/200157222-37d99b9e-8900-4186-9c2a-caa5113f9c4c.png)